spec/requests/api/v1.1/portfolios_controller_spec.rb

```diff
@@ -89,13 +89,6 @@
         expect(json["meta"]["count"]).to eq 2
         expect(json["data"].first["owner"]).to eq new_portfolio.owner
       end
-
-      it "does not sort when specified" do
```

Comment:
@lindgrenj6 This spec is not guaranteed to work since the order of objects coming back without sort cannot be determined. The spec fails intermittently

Reply:
@mkanoor Doesn't that get handled by the default scope?  https://github.com/RedHatInsights/catalog-api/blob/master/app/models/portfolio.rb#L14

Reply:
@gmcculloug When the sort is not specified and there are 2 records its possible that you might get [1,2] or [2,1] that order is not guaranteed.

Reply:
Should we just change this to then be a test about `.includes?` instead of just removing the whole thing?